vendor/assets/stylesheets/spree/frontend/global.css.scss

```diff
@@ -3815,7 +3831,9 @@ a.btn.btn-block {
 .pagination-wrap .btn-primary span {
 	position: relative;
 }
-
+.pagination-wrap .prev-page-icon span {

```

Comment:
Same comment for this as well. As per above

Reply:
sir because if we dont add these it was disturbing the design of previous page button on click as mentioned from mutayyab bhai

Reply:
I dnt agree with this point. Please discuss it with Umer. As we have all the design. We dnt need to add any extra class. If you need to add , then discuss it with me or Umer before implementation